From 6d315949e57dd6f7e3f04394989c03fb7360e659 Mon Sep 17 00:00:00 2001
From: warped-rudi <r.ihle@s-t.de>
Date: Fri, 22 Jul 2016 08:27:32 +0200
Subject: [PATCH 1/5] Do not hardcode location of config.py

Always load config.py from the addon's home directory to allow
installation in special://home/ as well as in special://xbmc/

Signed-off-by: warped-rudi <r.ihle@s-t.de>
---
 visualization.wavforhue/resources/settings.xml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/visualization.wavforhue/resources/settings.xml b/visualization.wavforhue/resources/settings.xml
index 55b6547..86843d3 100644
--- a/visualization.wavforhue/resources/settings.xml
+++ b/visualization.wavforhue/resources/settings.xml
@@ -18,16 +18,16 @@
   </category>
   <!--Setup SECTION-->
   <category label="30022">  
-    <setting id="config"         type="action"     label="30020" action="RunScript(special://home/addons/visualization.wavforhue/config.py,action=discover)"/>
+    <setting id="config"         type="action"     label="30020" action="RunScript($CWD/config.py,action=discover)"/>
     <setting id="hueBridgeIP"    type="ipaddress"  label="30000" enable="!eq(-1,true)" default="192.168.10.6"/>
     <setting id="hueBridgeUser"  type="text"       label="30024" enable="!eq(-2,true)" default="KodiVisWave"/>
     <setting 					           type="sep"/>
-    <setting id="config_lights"  type="action"     label="30027" action="RunScript(special://home/addons/visualization.wavforhue/config.py,action=config_lights)"/>
+    <setting id="config_lights"  type="action"     label="30027" action="RunScript($CWD/config.py,action=config_lights)"/>
     <setting id="activeLights"   type="text"       label="30004" default="1,2,3"/>
     <setting id="dimmedLights"   type="text"       label="30008" default="4,5"/>
     <setting id="afterLights"    type="text"       label="30019" default="4"/>
     <setting 					           type="sep"/>
-    <setting id="reset_settings" type="action"     label="30026" action="RunScript(special://home/addons/visualization.wavforhue/config.py,action=reset_settings)" option="close" />
+    <setting id="reset_settings" type="action"     label="30026" action="RunScript($CWD/config.py,action=reset_settings)" option="close" />
   </category>
   <!--Behavior SECTION-->
   <category label="30023">  
-- 
1.9.1

